service.autosubs/changelog.txt

```diff
@@ -0,0 +1,35 @@
+1.1.0
```

Comment:
still in the submission :) please drop

Reply:
you mean, that changelog.txt should not be commited? Who will make changes there? :)

Reply:
Yeah changelog is no longer used in kodi favor of the `<news>` tag in addon.xml. The reason we ask devs to drop from submission is to make the addon packages smaller (although usually not a great impact on the addon size).